docs/installation.md

```diff
@@ -3,6 +3,8 @@ Installation
 
 Notes on how to install
 
+##### Conda
+To install the core package run: `conda install -c conda-forge glidertools`.
 
 ##### PyPI
```

Comment:
Can you clean up the bit about PyPI install. Specificaly do we prefer that people use used Conda? Maybe say that "and to use pip do ...". Also, (I know this is old text), but can you update the sentence about the correct version of pip, if there is a correct version then which one is it?

Reply:
How is this?